pkg/provision/providers/qemu/launch.go

```diff
@@ -81,10 +67,10 @@ type LaunchConfig struct {
 	sdStubExtraCmdline       string
 	sdStubExtraCmdlineConfig string
 
-	// filled by CNI invocation
-	tapName string
-	VMMac   string
-	nsPath  string
+	// platform specific Network configuration
+	Network networkConfig
```

Comment:
this is just a suggesion, but we could probably make the diff smaller if simply embed the struct here, i.e. 
```suggestion
   networkConfig
```

Reply:
I kinda like it as is because the struct that has platform specific types is clearly separated from the rest